<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateLightDialog</class>
 <widget class="QDialog" name="CreateLightDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>160</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create Light</string>
  </property>

  <layout class="QVBoxLayout" name="rootLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>

   <!-- Title -->
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>28</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
background-color: #303645;
color: #a4b0bd;
font-weight: 600;
padding-left: 8px;
}</string>
     </property>
     <property name="text">
      <string>Create Light</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>

   <!-- Body -->
   <item>
    <layout class="QVBoxLayout" name="bodyLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>

     <!-- Name row -->
     <item>
      <layout class="QHBoxLayout" name="nameRow">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="nameLabel">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit">
         <property name="placeholderText">
          <string>Light</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>

     <!-- NEW: Name hint row -->
     <item>
      <widget class="QLabel" name="nameHintLabel">
       <property name="styleSheet">
        <string notr="true">QLabel { color: #6b7580; padding-left: 88px; }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>

     <!-- Type row -->
     <item>
      <layout class="QHBoxLayout" name="typeRow">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="typeLabel">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typeCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed"/>
         </property>
         <item>
          <property name="text">
           <string>Directional</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Point</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Spot</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>

     <!-- Spacer -->
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>

     <!-- Buttons -->
     <item>
      <layout class="QHBoxLayout" name="buttonRow">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <spacer name="buttonSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="createButton">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Create</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>

    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
